tests/astrapy/test_db_dml.py

```diff
@@ -456,8 +462,15 @@ def test_concurrent_chunked_insert_many(
     ]
 
     with pytest.raises(ValueError):
+        # the first doc must be pre-existing
+        # and the doc array size must be <= chunk size
+        # for this not to spoil the rest of the test
+        docs_for_error = documents0[0:1] + [{"_id": str(uuid.uuid4())}]
         _ = writable_vector_collection.chunked_insert_many(
-            documents1, chunk_size=3, concurrency=4
+            docs_for_error,
+            chunk_size=3,
+            concurrency=4,
+            options={"ordered": True},
```

Comment:
I wonder if it makes sense to allow to have both concurrency > 1 and ordered=True.
It might not give what the user expects...

Reply:
Good point, I've been thinking about that a few times. But I would not go farther than issuing a warning if that condition is detected, wdyt? Being too prescriptive might get outdated faster possibly